apps/site/assets/css/_utilities.scss

```diff
@@ -213,6 +213,14 @@
   color: $gray;
 }
 
+.u-color-brand-primary-light {
+  color: $brand-primary-light;
+}
+
+.u-color-brand-primary {
+  color: $brand-primary;
+}
+
```

Comment:
FYI the `text-primary` class is equivalent (thank Bootstrap).

Reply:
Good to know 😅.  Its hard to find documentation for what our current version of bootstrap supports

Reply:
It really is hard... Bootstrap doesn't host the docs for our version anywhere as far as I know, so I've had to look at the [codebase at that version tag](https://github.com/twbs/bootstrap/tree/v4.0.0-alpha.2) many a time 😵